components/datetime/src/time_zone.rs

```diff
@@ -193,27 +203,152 @@ impl TimeZoneFormat {
                 TimeZone::UpperV => match length {
                     1 => (), // BCP-47 identifier, no CLDR-data necessary.
                     2 => (), // IANA time-zone ID, no CLDR data necessary.
```

Comment:
Not necessarily.

For `V`, the UTS-35 spec specifies:
> The short time zone ID. Where that is unavailable, the special short time zone ID unk (Unknown Zone) is used.

For `VV` there is no fallback specified.

The spec doesn't say anything about falling back to GMT or otherwise in these cases.

But if we make the `fallback_unit` field non-optional, defaulting to GMT, it will be set regardless.

Reply:
My feeling is that `fallback_unit` is non-optional, defaulting to GMT, and users who want no-fallback behavior should use the no-fallback function. But, I'm not sure what that means with regards to spec compliance.

Reply:
@samchen61661 this looks to be resolved now, correct?

Reply:
i think so. Setting the `fallback_unit` is within `TimeZoneFormat::get_fallback_unit(options.fallback_format)` while initializing a `TimeZoneFormat` struct. In other words, these cases will have default fallback as well by calling the fallback function.